src/data_import/database_reader.py

```diff
@@ -17,7 +17,8 @@ def __init__(self, db_name: str, coll_name: str):
             # TODO get columns name of collection
             self.columns_name = list(self.database.get_nth_chunk('', []).columns)
```

Comment:
it's done, if I'm not mistaken?

Reply:
Imo no, in reader is simpler function get_columns_names now